console.lua

```diff
@@ -187,6 +187,73 @@ commands = {{
         end
         return "Discards are now " .. G.GAME.current_round.discards_left
     end
+}, {
+    name = "joker",
+    source = "debugplus",
+    shortDesc = "Spawns a joker",
+    desc = [[Spawns Jokers. Usage:\njoker {args} [name]\n 
+            -a : Skip calling add_to_deck on spawned joker
+            -c : Count of jokers to spawn
+            -e : Edition to spawn jokers with
+            -E : enable jokers spawned to have random editions
+        ]],
+    exec = function(args, rawArgs, dp)
+        if G.STAGE ~= G.STAGES.RUN then
+            return "This command must be run during a run.", "ERROR"
+        end
+        local j_key = nil
+        local j_count = 1
+        
+        local j_no_edition = true
+        local j_set = "Joker"
+        local j_edition = nil
+        local j_rarity = nil
+        local add_to_deck = true
+
+        -- I thought of making a full on arg parser for this command
+        -- that started as a meme so I could spawn 1000 copies of jimbo without using eval
+        local skipNextArg = false
+        for i,v in ipairs(args) do 
+            if skipNextArg then
+                skipNextArg = false
+                -- Don't parse args to other parm
+            elseif not (v:sub(1,1) == "-" ) then
+                j_key = "j_" .. v
```

Comment:
Perhaps check if the user already put a j_ there.

There also doesn't seem to be any check for if the joker key is a valid joker. Can probably just steal the check from https://github.com/WilsontheWolf/DebugPlus/blob/f1b348b6c22edc6da8afbc65ab709beb63f0b678/watcher.lua#L199-L207.